web/cobrands/sass/_base.scss

```diff
@@ -2682,6 +2660,49 @@ label .muted {
   }
 }
 
+.postcode-form-box {
+  margin: 0 -1em;
+  padding: 1em;
+  color: $front-main-color;
+  background: $postcodeform-background;
+  font-family: $meta-font;
+  label {
+    margin-top: 0;
```

Comment:
Do you have an example where this makes a difference?

Reply:
Yes, top of my mind Camden and Bromley